### PR TITLE
Backported Storage Downgrade

### PR DESCRIPTION
**Pull request by johnch18**
I backported this from the 1.12 version of the mod.
It sets the base storage of a drawer to 1 stack, useful for creating inventories with a specific number of stacks.
e.g. A drawer with a downgrade and two emerald upgrades would have the normal number of stacks.

### CODE REVIEW

File: src/com/jaquadro/minecraft/storagedrawers/block/tile/TileEntityDrawers.java [hunk 1]
```diff
@@ -5,10 +5,7 @@
 import com.jaquadro.minecraft.storagedrawers.api.security.ISecurityProvider;
 import com.jaquadro.minecraft.storagedrawers.api.storage.IDrawer;
 import com.jaquadro.minecraft.storagedrawers.api.storage.IDrawerGroupInteractive;
-import com.jaquadro.minecraft.storagedrawers.api.storage.attribute.ILockable;
-import com.jaquadro.minecraft.storagedrawers.api.storage.attribute.IProtectable;
-import com.jaquadro.minecraft.storagedrawers.api.storage.attribute.ISealable;
-import com.jaquadro.minecraft.storagedrawers.api.storage.attribute.LockAttribute;
+import com.jaquadro.minecraft.storagedrawers.api.storage.attribute.*;
```
Comment (mitchej123) [inline src/com/jaquadro/minecraft/storagedrawers/block/tile/TileEntityDrawers.java:8]: nit: Not a fan of .* imports; I'd prefer explicit imports.  (It's an IDE option you can set too)

File: src/com/jaquadro/minecraft/storagedrawers/core/ModItems.java [hunk 2]
```diff
@@ -50,6 +53,9 @@ public void init () {
             GameRegistry.registerItem(personalKey, "personalKey");
         if (StorageDrawers.config.cache.enableTape)
             GameRegistry.registerItem(tape, "tape");
+        if (true) {
```
Comment (mitchej123) [inline src/com/jaquadro/minecraft/storagedrawers/core/ModItems.java:56]: rm

**Review by mitchej123 — CHANGES_REQUESTED** [review]
Mostly looks fine.  I'm not seeing an icon for the downgrade, I'd like to see that added, and some screenshots with evidence everything is working as expected.

File: src/com/jaquadro/minecraft/storagedrawers/core/ModItems.java (hunk 2 above)
Comment (johnch18) [reply src/com/jaquadro/minecraft/storagedrawers/core/ModItems.java:56]: It's a placeholder for an actual config check

Comment (mitchej123) [reply src/com/jaquadro/minecraft/storagedrawers/core/ModItems.java:56]: Implement the config check or remove it :P

Comment (johnch18) [reply src/com/jaquadro/minecraft/storagedrawers/core/ModItems.java:56]: Yessir :(

**Comment by johnch18** [comment]
There we go. Had some issues with IntelliJ that I had to pin down.
image

**Review by mitchej123 — APPROVED** [review]

**Comment by Prometheus0000** [comment]
@johnch18 You added the recipe for default, but not for GTNH, so there is no recipe. See https://github.com/GTNewHorizons/StorageDrawers/pull/3/files.

**Pull request merged**